SECURITY.md

```diff
@@ -1,16 +1,19 @@
-# Security Policy
+# 👮‍♀️ Security Policy
 
-## Supported Versions
+This document describes the security policy of the Registry API Service.
 
-Use this section to tell people about which versions of your project are
-currently being supported with security updates.
+
+## 📀 Supported Versions
+
+The following table describes what versions are supported and which are not.
 
 | Version | Supported          |
-| ------- | ------------------ |
-| 1.18.2   | :white_check_mark: |
-| 1.17.5   | :white_check_mark: |
-| < 1.17.5   | :x: |
+|:--------|:------------------:|
+| 0.3.2   | ✅                 |
+| 0.3.0   | ❌                 |
+| <0.3    | ❌                 |
+
 
-## Reporting a Vulnerability
+## 🚨 Reporting a Vulnerability
 
-Please create a new issue using the [Vulnerability Issue template](https://github.com/NASA-PDS-Incubator/validate/issues/new?assignees=jordanpadams&labels=triage-needed%2C+security&template=vulnerability-issue.md&title=%5BSECURITY%5D+Title+Here)
+Please create a new issue using the [Vulnerability Issue template](https://github.com/NASA-PDS/registry-api-service/issues/new?assignees=jordanpadams&labels=bug%2C+needs%3Atriage%2C+security&template=vulnerability-issue.md&title=%5BSECURITY%5D+Title+Here).
```

Comment:
@nutjob4life for the security policy, I would recommend we instead just point people to https://github.com/NASA-PDS/registry-api-service/issues/new versus the specific template because the link has the template embedded in the URL. so if we update the template in the repo, those changes will not make it into these tickets (if that makes sense?).

Reply:
Sounds good to me